tests/backend/common.py

```diff
@@ -199,7 +200,12 @@ def __init__(self, cmd, serializer, parse_args=lambda self: {}, check_rep_by_def
     async def _do_send(self, ws, req_post_processing, args, kwargs):
         req = {"cmd": self.cmd, **self.parse_args(self, *args, **kwargs)}
         if req_post_processing:
-            raw_req = packb(req_post_processing(self.serializer.req_dump(req)))
+            if not isinstance(self.serializer, ApiCommandSerializer):
+                pre_processed_req = req_post_processing(self.serializer.req_dump(req))
+                raw_req = packb(pre_processed_req)
+            else:
+                pre_processed_req = req_post_processing(req)
+                raw_req = self.serializer.req_dumps(req)
```

Comment:
What have changed ?

Reply:
This is because of the `req_post_processing` callback we can't call `packb` on rust object :(

Reply:
Note that tests commands are sent as legacy `dict` https://github.com/Scille/parsec-cloud/blob/a5a95ccc7c0236c350c415add9cbaae90f5ca58c/tests/backend/common.py#L648 (as it's done for every commands)

Reply:
That's because of `tests/backend/test_pki.py::test_pki_submit_no_email_provided` which make use of this callaback to ensure no email address is being sent to the server (which is the purpose of the test).